server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/DomainQuotaRoutesTest.java

```diff
@@ -43,16 +43,16 @@
 class DomainQuotaRoutesTest {
 
     private static final String QUOTA_DOMAINS = "/quota/domains";
-    private static final String PERDU_COM = "perdu.com";
-    private static final Domain TROUVÉ_COM = Domain.of("trouvé.com");
+    private static final String LOST_COM = "lost.com";
+    private static final Domain FOUND_COM = Domain.of("found.com");
```

Comment:
trouvé.com and perdu.com are real very simple html site, while found.com and lost.com seems to be heavy commercial site, which I would prefer not to link to

Reply:
Didn't even think about that oO just did a silly on the fly translation... Will check that :)

Reply:
It's tests after, shouldn't probably use existing domain names at all